zadani/soft/style.css

```diff
@@ -11,33 +13,166 @@
 body {
     margin: 0;
     padding: 0;
-    background-color: #f5f5f5;
     font-family: 'Merriweather', serif;
     font-size: 16px;
     line-height: 1.6;
+    background-color: #f5f5f5;
 }
 
-
 /* HLAVICKA */
 
+.hlavicka {
+    display: flex;
```

Comment:
Display bychom klidně mohli nechat výchozí `block`, protože hlavička je ve všech zobrazeních (mobil, tablet, desktop) stejná, tj. vše pod sebou a nebudeme to potřebovat v media queries měnit. Ale flex ničemu neškodí, jen to není nejminimalističtější řešení 🙂